src/styles/header.scss

```diff
@@ -0,0 +1,78 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  .logo-container {
+    width: 40px;
+    height: 40px;
+
+    .logo {
+      display: block;
+      width: 100%;
+      height: auto;
+    }
+  }
+
+  .nav {
+    display: flex;
+    align-items: center;
+    transition: color 0.3s ease;
+
+    .nav__item {
+      margin-right: 20px;
+      height: 60px;
+      line-height: 60px;
+      padding: 0;
+
+      &:last-child {
+        margin-right: 0;
+      }
+
+      .nav__link {
+        text-decoration: none;
+        color: $nav-link-color;
+        height: 60px;
+
+        &:hover {
+          color: $blue-color;
+        }
+
+        &.is-active {
+          color: $blue-color;
+
+          &::after {
+            content: '';
+            height: 4px;
+            display: block;
+            background-color: $blue-color;
```

Comment:
The variable `$blue-color` is used here but not defined. Please ensure that this variable is defined in a `vars.scss` file or imported correctly to avoid compilation errors.